pkg/pcr/get_measurements_pcr0_cbnt0t.go

```diff
@@ -71,33 +73,63 @@ func MeasurePCR0Data(config MeasurementConfig, imageSize uint64, fitEntries []fi
 		Length: uint64(len(acmEntry.GetRSASig())),
 	}
 
-	keyManifest, keyManifestFITEntry, err := getKeyManifest(fitEntries)
+	keyManifest1, keyManifest2, keyManifestFITEntry, err := getKeyManifest(fitEntries)
```

Comment:
These are different types of manifests and it would be good if the variables they are assigned to could reflect this rather than just having a 1/2 suffix (same comment applies to the later calls to `getKeyManifest`)

Reply:
Change a naming a bit (and added comments), let me know if not good enough.